Source/RapyutaSimulationPlugins/Public/Core/RRTypeUtils.h

```diff
@@ -17,9 +17,27 @@
 #include "RRTypeUtils.generated.h"
 
 /**
- * @brief UE type related utils
+ * @brief UE/std type related utils
  * 
  */
+template<typename T>

```

Comment:
whrere those struct are used?

Reply:
It is used [here](https://github.com/rapyuta-robotics/RapyutaSimulationInternal/blob/AddDataSynth/Source/RapyutaSimInternal/Public/SceneCapture/RRCaptureCommon.h#L318) in `URRCaptureCommon::LogJsonProp`